cmd/influx/bucket.go

```diff
@@ -320,7 +328,7 @@ func (b *cmdBucketBuilder) printBuckets(printOpt bucketPrintOpt) error {
 
```

Comment:
It's too far from the diff for me to comment on directly, but I think we'll also need support in the `update` command.

The server-side code for the update path will also need changed. At the moment, I believe it will reset the shard-group duration every time the retention policy is updated. We probably instead want the `update` API to accept both an RP and a shard-group duration (each one optional), and use:

- Duration is unset = don't change the current shard-group duration
- Duration set to 0 = have the server recompute a "good" duration based on the bucket's RP